apps/explorer/priv/repo/migrations/20200922134522_token_transfer_more_indices.exs

```diff
@@ -0,0 +1,63 @@
+defmodule Explorer.Repo.Migrations.TokenTransferMoreIndices do
+  use Ecto.Migration
+
+  def up do
+    execute("""
+    DELETE FROM token_transfers WHERE log_index >= 1000000;
+    """)
+
+    create(index(:token_transfers, ["block_number DESC, amount DESC, log_index DESC"]))
+
+    create(
+      index(:token_transfers, ["block_number DESC, transaction_hash DESC, from_address_hash DESC, to_address_hash DESC"])
+    )
+
+    execute("""
+    INSERT INTO token_transfers (
+      SELECT hash, block_hash, -(index*1000+1000000) AS log_index, from_address_hash, to_address_hash, value, null,
+          tokens.contract_address_hash, tx.inserted_at, tx.updated_at, block_number, null
+      FROM transactions AS tx, tokens
+      WHERE value > 0 AND tokens.symbol = 'cGLD' AND to_address_hash IS NOT NULL
+    );
+    """)
+
+    execute("""
+    INSERT INTO token_transfers (
+      SELECT transaction_hash, block_hash, -(index+transaction_index*1000+1000000) AS log_index, from_address_hash, to_address_hash, value, null,
```

Comment:
Then we need to update both `DELETE` statements as well.

Reply:
good catch, done